modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -2245,6 +2554,25 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private Long _generateRandomDocument() throws Exception {
```

Comment:
I think it would be more appropriate replace with _add[...]. See the patterns:

```
>> git grep "private .* _add" *Test.java | wc -l
785
```

```
>> git grep "private .* _generate" *Test.java | wc -l
29
```

Reply:
It is weird to call the method `_generateRandomDocument` and then, return an `id`. That does not match the pattern. What about returning a Document and then, using the id?